<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>350</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>현대하모니 M</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>AWND</string>
  </property>
  <widget class="QPushButton" name="Input_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>플랫폼</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sheet Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>실행</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>421</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Input_button_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>데이터 생성</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 B</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Input File 을 넣고 데이터 생성을 누르세요!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sheetSelect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>시트 선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="platformSelect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>플랫폼 선택</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Input File</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Create Init Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="arrowLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;&lt;--</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>20</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="wideButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>16</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Additional Option</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="climateCheck">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CLIMATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Domain</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="absCheck">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Domain 독립생성</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>272</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Input_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OK_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Input_button_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_data_gen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sheetSelect</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_sheet_select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>platformSelect</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_platform_select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wideButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_wide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>absCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>slot_abs_domain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_button()</slot>
  <slot>slot_platform()</slot>
  <slot>slot_sheet()</slot>
  <slot>slot_start()</slot>
  <slot>slot_data_gen()</slot>
  <slot>slot_exec()</slot>
  <slot>slot_cancel()</slot>
  <slot>slot_sheet_select()</slot>
  <slot>slot_platform_select()</slot>
  <slot>slot_wide()</slot>
  <slot>slot_abs_domain()</slot>
  <slot>slot_reset()</slot>
 </slots>
</ui>
